### DISCUSSION TITLE
LEDのランド剥がれ

### DISCUSSION THREAD

**Discussion by ntnl-it-wiz** [Yowkees/keyball#465] [Q&A] [answered]
ダメもとで伺いますが、LEDのランドをうっかり剝がしてしまい、よいリカバリー方法があればご教示いただきたく、相談いたします。
![image](https://github.com/Yowkees/keyball/assets/42383560/3fa45f39-5b25-4b47-b6ab-afeb551f4945)

3のDOUT部分が剝がれてしまい、4のDINと導通ができない状況です。
単純な方法としては該当箇所をエナメル線でつなげばよいかとは思いますが、少し離れていることもあり、他によいリカバリー方法があればご教示いただけますと幸いです。

**Comment by ntnl-it-wiz** [comment]
素案として、3 DOUTの裏面と表面のランド、4 DINの裏面と表面のランドをエナメル線で結ぶと、結線が目立たないかなと思ったのですが、不都合ありますでしょうか。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

回答が遅くなり申し訳ありません。
ご提案の通り、裏面のランドが使えると思います。LED4は触る必要ありません。
LED3の剥がれてしまったランド部のLED端子を下記写真の”裏面”の青印をつけた端子に接続できれば復活できます。

お手数ですがご対応お願いいたします。
![KB61_LED3fix](https://github.com/Yowkees/keyball/assets/84209772/b096f8ab-cf38-44aa-a997-84cf35b515ef)

**Reply by ntnl-it-wiz** [reply]
回答ありがとうございます。こちらもその後時間が取れず返信が遅くなり申し訳ありません。
裏と表でエナメル線を連結して3のDOUTと4のDINの導通が取れるようになり、安心したのも束の間、4のLEDが点灯しません。
4のLEDは2回交換、3のLEDも1回交換してダメなので個体不良ではなく別の原因ではないかと思われます。
この場合、どこを確認するべきでしょうか。

ついでに、左側のLEDも37だけ点灯しないのですが、37の場合どこと導通確認するべきかご教示いただけますでしょうか。
（1個だけなのでこちらは最悪諦めがつくかなと思っています…。

**Comment by Yowkees** [comment]
ご対応ありがとうございます。

3のDOUTと4のDINが導通しているようですので、あとはVCCもしくはGNDの半田付け不良がないかご確認ください。
LEDはとても湿気に弱いので、開封後すぐに半田付けすることと、半田こて先を270℃以下にして作業することをお勧めします。

下記リンクにてKeyball61の基板データを公開しています。KiCAD等フリーソフトで表示していただいて線を追っていただくと、LEDの繋がりが把握できると思いますのでご確認いただければ幸いです。
https://github.com/Yowkees/keyball/tree/main/keyball44/design_data

左側37番のLEDも１つだけ不良ということで、VCCかGND端子の半田付けを再度ご確認いただければ幸いです。

以上、お手数ですがご確認お願いいたします。

**Reply by ntnl-it-wiz** [reply]
すみません、言葉足らずでした。
LED4について、VCC同士での導通確認、GND同士での導通確認ができております。
よって、それ以外の要因ではないかと思いまして、他に確認するべきはないかという質問をさせていただきました。
例えば以前別のLEDで、1つ前のDOUTとGNDが短絡しており通電できなかったことがありました。
今回はLED3側の短絡がないことも確認しております。
LED4そのもの以外で再確認すべき箇所や、切り分け方法、他のリカバリー手段など何かないでしょうか。
現状では全ての疎通が通っており、新品のLEDへの付け替えのし直しも試して駄目なのでお手上げ状態です。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

LED5は半田付けしていない状態だとすると、LED3のDOUTとLED4のDINの導通、あとはVCCどうしGNDどうしの導通、この３本の線だけ繋がれば確実に点灯すると思います。
この３線の確認以外には切り分け方法等はないと思います。

LEDは湿気っていると加熱後すぐに故障してしまいますので、新鮮なものを使用することをお勧めします。

以上、お手数ですが再度ご確認をお願いいたします。

**Comment by ntnl-it-wiz** [comment]
![image](https://github.com/Yowkees/keyball/assets/42383560/2373782c-2307-4146-bcb1-bb2340cb0dd4)
その後、2本半田ごてで強めに抑え込んだりしたのが功を奏したのか、点灯するようになりました。
4番以降はそこまで時間が掛かることはなく、2つ目のKeyball61も無事に完成いたしました。ご回答ありがとうございました。
